lib/wallaroo/core/sink/connector_sink/connector_sink.pony

```diff
@@ -1488,3 +1543,109 @@ class PauseBeforeReconnectConnectorSink is TimerNotify
     _tcp_sink.reconnect()
     Backpressure.release(_auth)
     false
+
+trait SinkValve
```

Comment:
Can you document what this is, why it exists, and roughly how it works?

Reply:
Updated by commit db904d6